tmt/export/polarion.py

```diff
@@ -217,7 +226,10 @@ def export_to_polarion(test: tmt.base.Test) -> None:
         polarion_case.caselevel = 'component'
         polarion_case.testtype = 'functional'
         if test.component:
-            polarion_case.casecomponent = [test.component[0]]
+            try:  # depending on project this can require either single item or list
+                polarion_case.casecomponent = test.component[0]
+            except PolarionException:
+                polarion_case.casecomponent = [test.component[0]]
```

Comment:
If the project supports multiple components, shall we sync all components instead of just the first one?
```suggestion
                polarion_case.casecomponent = test.component
```

Reply:
I'll rather have consistent behaviour across all projects in always taking the first component. Haven't really met any test using multiple components yet. Will add a comment.

Reply:
Ok, sounds good.